website/static/js/myProjects.js

```diff
@@ -1777,31 +1777,7 @@ var Filters = {
  */
 var Information = {
     view : function (ctrl, args) {
-        function categoryMap(category) {
-            // TODO, you don't need to do this, CSS will do this case change
-            switch (category) {
-                case 'analysis':
-                    return 'Analysis';
-                case 'communication':
-                    return 'Communication';
-                case 'data':
-                    return 'Data';
-                case 'hypothesis':
-                    return 'Hypothesis';
-                case 'methods and measures':
-                    return 'Methods and Measures';
```

Comment:
I guess this is a case of two-birds-one-stone, changing to it text-capitalize stops it from capitalizing 'and', I think we're ok with that.